tuned.spec

```diff
@@ -264,6 +264,17 @@ Requires: %{name} = %{version}
 %description profiles-openshift
 Additional TuneD profile(s) optimized for OpenShift.
 
+%package ppd
+Summary: PPD compatibility daemon
+Requires: %{name} = %{version}
+# The compatibility daemon conflicts with PPD,
+# since it exports the same DBus object.
+Conflicts: power-profiles-daemon
```

Comment:
This is fine, but we probably want to adjust both `power-profiles-daemon` and `tuned-ppd` packaging to Provide+Conflicts so that it becomes swappable. Maybe something like:

```
Provides: ppd-service
Conflicts: ppd-service
```

And we'll need to get consumers of `power-profiles-daemon` to require `ppd-service` instead.